--- saved/adopt.txt
+++ live/adopt
@@ -29,19 +29,23 @@
 - Some are released to us by their owners due to changes in family circumstances
 - Some are from owners realizing that a Chihuahua is not a good match for them
 - Some are strays from animal shelters and humane societies
-- A few come from puppy mills and backyard breeders.
+- A few come from puppy mills and backyard breeders
 
-Our adult Chihuahuas are spayed/neutered, tested and if necessary, treated, for heartworm and other parasites. All Chihuahuas are brought up to date on their inoculations prior to their placement in loving forever homes.
+Our adult Chihuahuas are spayed/neutered, tested and, if necessary, treated for heartworm and other parasites. All Chihuahuas are brought up-to-date on their inoculations prior to their placement in loving forever homes.
 
 Adoption Process
 
-The first step in adoption is to fill out anAdoption ApplicationThis form gives basic information about the type of Chihuahua that will best fit into your family. Once the application is received and reviewed to ensure it is complete and meets our adoption criteria, a YCRAA volunteer will then contact you for a phone interview and arrange for a home visit. When there is a particular YCRAA dog you may be interested in then the Foster Mom will also contact you to talk about their foster. We do not have a shelter or facility where the Chihuahuas may be visited, we are a network of foster homes here in New England. Foster parents volunteer their time and put a lot into evaluating and rehabilitating YCRAA Chihuahuas until they are ready for placement.
+The first step in adoption is to fill out an
+
+Adoption Application
+
+This form gives basic information about the type of Chihuahua that will best fit into your family. Once the application is received and reviewed to ensure it is complete and meets our adoption criteria, a YCRAA volunteer will then contact you for a phone interview and arrange for a home visit. When there is a particular YCRAA dog you may be interested in, then the foster mom will also contact you to talk about their foster. We do not have a shelter or facility where the Chihuahuas may be visited, we are a network of foster homes here in New England. Foster parents volunteer their time, and put a lot into evaluating and rehabilitating YCRAA Chihuahuas until they are ready for placement.
 
 If it seems like a good match then a visit is arranged to meet the dog. Sometimes a home is approved to adopt a YCRAA Chihuahua but it is not the best match for the particular dog requested. The application then remains on file as a "Pre-approved Home" until the right dog comes along. We often have dogs that have just come in and are not listed yet, but the Adoption Coordinator will know about them. Many of our YCRAA Chihuahuas get matched to applicants that are waiting on our "Approved Homes" list. Becoming an Approved Home is the best way to be matched with a YCRAA dog.
 
 Cost of an Adoption (donation plus applicable medical)
 
-Adoption cost is based on age (donation)plus any/all applicable medical expenses.YCRAA willingly accepts Chihuahuas and Chi mixes young and old. Any one dog may need expensive veterinary treatment to fulfill quality of life standards. Adoption donations do not cover any veterinary/medical costs. Be aware that there may be additional medical expenses added to the adoption donation as the final cost for adoption.
+Adoption cost is based on age (donation)plus any/all applicable medical expenses.YCRAA willingly accepts Chihuahuas and Chi mixes young and old. A dog may need expensive veterinary treatment to fulfill quality of life standards. Adoption donations do not cover any veterinary/medical costs. Be aware that there may be additional medical expenses added to the adoption donation as the final cost for adoption.
 
 When dogs are not suitable for adoption, then they live out their lives with a volunteer as part of our "Special Needs" program. We depend on private donations and the sale of items from our web site to supplement the adoption donations.
 
@@ -54,11 +58,11 @@
 |  | 4 - 6 years | $250 | To be determined if any
 |  | 6 - 8 years | $200 | To be determined if any
 |  | 8 - 10 years | $100 | To be determined if any
-|  | over 10 years | $ 75 or negotiable to a great home | To be determined if any
+|  | over 10 years | $75 or negotiable to a great home | To be determined if any
 
-For puppies, YCRAA requires adopters to sign a spay/neuter contract and to submit a pre-paid receipt from a veterinarian for spay/neuter surgery (adopter pre-paid expense).
+For puppies, YCRAA requires adopters to sign a spay / neuter contract and to submit a prepaid receipt from a veterinarian for spay / neuter surgery (adopter prepaid expense).
 
-For older dogs, e.g. Chihuahuas with a known health issue, then the donation amount requested may be reduced but still may include applicable medical expenses e.g. medications that go with the adoption.
+For older dogs (e.g., Chihuahuas with a known health issue) then the donation amount requested may be reduced, but still may include applicable medical expenses, e.g., medications that go with the adoption.
 
 Adoption Policies of YCRAA
 
@@ -69,8 +73,8 @@
 - Contact Chihuahua Rescue and Transport (CRT) for information about adoption in your area.
 - YCRAA offers dogs for adoption only to households in which there are no children under 10 years of age without special approval of the YCRAA Board of Directors.
 - All adult dogs are neutered before placement.
-- YCRAA offers dogs for adoption only to households in which there are NO intact dogs. For more information on our policy;Why we do not place dogs in homes with intact animals.
-- All dogs under 6 months require a signed Spay/Neuter Contract and a signed receipt from the adopter's veterinarian stating that spay/neuter surgery has been paid in full by the adopter.
+- YCRAA offers dogs for adoption only to households in which there are NO intact dogs. For more information on our policy:Why we do not place dogs in homes with intact animals.
+- All dogs under 6 months require a signed Spay / Neuter Contract and a signed receipt from the adopter's veterinarian stating that spay / neuter surgery has been paid in full by the adopter.
 - All dogs are microchipped and registered with YCRAA as the primary contact.
 
 Come see our Available Dogs
@@ -79,24 +83,24 @@
 
 We are often asked, "Why will you not place a Yankee Chihuahua in a home with dogs that have not been neutered?"
 
-The main reason we will not place a Yankee Chihuahua in a home where the current pets have not been neutered, is because we want to place our Chihuahuas in homes with responsible owners. Responsible owners take good care of their pets and do the best thing for them -- which includes Neutering.
+The main reason we will not place a Yankee Chihuahua in a home where the current pets have not been neutered, is because we want to place our Chihuahuas in homes with responsible owners. Responsible owners take good care of their pets and do the best thing for them — which includes neutering.
 
 There are several health benefits to neutering.
 
-One of the most important concerns the prostate gland, which under the influence of testosterone will gradually enlarge over the course of the dog's life. In age, it is likely to become uncomfortable, possibly being large enough to interfere with defecation. The prostate under the influence of testosterone is also predisposed to infection which is almost impossible to clear up without neutering. Neutering causes the prostate to shrink into insignificance thus preventing both prostatitis (an infection, usually caused by bacteria) as well as the uncomfortable benign hyperplasia (enlargement) that occurs with aging. It is often erroneously held that neutering prevents prostate cancer but this is not true.
+One of the most important concerns the prostate gland, which under the influence of testosterone will gradually enlarge over the course of the dog's life. In age, it is likely to become uncomfortable, possibly being large enough to interfere with defecation. The prostate under the influence of testosterone is also predisposed to infection, which is almost impossible to clear up without neutering. Neutering causes the prostate to shrink into insignificance thus preventing both prostatitis (an infection, usually caused by bacteria) as well as the uncomfortable benign hyperplasia (enlargement) that occurs with aging. It is often erroneously held that neutering prevents prostate cancer, but this is not true.
 
 Other health benefits of neutering include the prevention of certain types of hernias and tumors of the testicles and anus. Excessive prepuce/penis discharge is also reduced by neutering.
 
-Thebehaviorof other dogs in homes our fosters go into is a huge concern of YCRAA. Un-neutered dogs have natural biological needs. If the dog doesn't fulfill those needs, he will build up frustration, anxiety, and may even develop obsessive tendencies.
+Thebehaviorof other dogs in homes our fosters go into is a huge concern of YCRAA. Unneutered dogs have natural biological needs. If the dog doesn't fulfill those needs, he will build up frustration, anxiety, and may even develop obsessive tendencies.
 
-Un-neutered dogs often are:
+Unneutered dogs often are:
 
 - Wanderers & roamers
-- Noisy, Baying, howling, overbearing, barking & lunging at passersby
+- Noisy, baying, howling, overbearing, barking & lunging at passersby
 - Tend toward fighting, lunging & barking at and fighting with other male dogs
-- Non-compliant; pushy and bossy attitude towards caretakers and strangers
-- Uncooperative, Resistant; an unwillingness to obey commands
-- Excessive sniffers have sexual frustration; mounting other dogs, people, toys
+- Non-compliant, pushy and bossy attitude towards caretakers and strangers
+- Uncooperative, resistant — an unwillingness to obey commands
+- Excessive sniffers having sexual frustration; mounting other dogs, people, toys
 - Excessive grooming of genital area
 - Offensive growling, snapping, biting
 - A heightened sense of territoriality, marking with urine indoors
@@ -105,9 +109,9 @@
 
 Not the type of circumstances we want our fosters to go into.
 
-The only behavior changes after neutering relate to behaviors influenced by male hormones. Playfulness, friendliness, and socialization with humans are not changed. Activity level and appetite do not change with neutering. A male dog should not gain weight or become less interested in activity post neuter.
+The only behavior changes after neutering relate to behaviors influenced by male hormones. Playfulness, friendliness, and socialization with humans are not changed. Activity level and appetite do not change with neutering. A male dog should not gain weight or become less interested in activity post-neuter.
 
-With Neutering, the interest in roaming is eliminated in 90 percent of neutered dogs. Aggressive behavior against other dogs is eliminated in 60 percent of neutered dogs. Urine marking is eliminated in 50 percent of neutered male dogs. Inappropriate mounting is eliminated in 70 percent of neutered dogs. In other words, they are much better behaved.
+With neutering, the interest in roaming is eliminated in 90 percent of neutered dogs. Aggressive behavior against other dogs is eliminated in 60 percent of neutered dogs. Urine marking is eliminated in 50 percent of neutered male dogs. Inappropriate mounting is eliminated in 70 percent of neutered dogs. In other words, they are much better behaved.
 
 Well-behaved companions are what a Yankee Chihuahua deserves.
 
@@ -115,7 +119,7 @@
 
 Others are casualties of a divorce or a move to a place where dogs are not allowed. They are often frightened at first, and confused by the loss of a long-familiar home. It can take a little extra time and patience to convince them that they can be safe and loved again. Once they feel secure, though, they seem to know they've been given a second chance at life. They don't ask for much: just a warm place to sleep, good meals, and plenty of love. They will not be with you as long as a younger dog - but you will have given them years of love in place of fear, confusion, and a premature death. And the love and gratitude they give in return is a very special thing.
 
-If you would consider adopting an older dog, please fill in an application. Often these wonderful dogs need a new home quickly. If we have your application and are on our waiting list they can be adopted right to you and the trauma of losing their home is greatly reduced.
+If you would consider adopting an older dog, please fill out an application. Often these wonderful dogs need a new home quickly. If we have your application and are on our waiting list they can be adopted right to you and the trauma of losing their home is greatly reduced.
 
 Reasons to Adopt a Senior Dog
 
@@ -125,7 +129,7 @@
 - When you rescue a senior dog, you have a best friend for life.
 - Senior dogs reward your care with an unwavering devotion.
 - Nothing matches the love of a senior dog for his family.
-- Senior dogs have learned many of life's lessons. They know, for example, that shoes are for walking and bones are for chewing. Senior dogs know that great outdoors is for eliminating and the house is for relaxing. Your carpet will last longer with a senior dog.
+- Senior dogs have learned many of life's lessons. They know, for example, that shoes are for walking and bones are for chewing. Senior dogs know that the great outdoors is for eliminating and the house is for relaxing. Your carpet will last longer with a senior dog.
 - Senior dogs can learn new tricks and be valuable family and community members.
 - They make excellent therapy dogs.
 - Senior dogs often fit into your household with ease.
@@ -135,7 +139,7 @@
 - Passed over for cute and cuddly puppies, they often do not have a chance and must go to make space for more puppies.
 - Adopting a senior dog saves a life!
 
-Come see our Available dogs.
+Come see our Available dogs
 
 If you experience problems with the online form please request an Adoption Application by email to:
 
@@ -144,7 +148,7 @@
 
 Thanks so much!
 
-For more information having anything to do with Yankee Chihuahua Rescue And Adoption, Inc., please find your state below, click on it, and send a letter to your state coordinator. You may write concerning adoptions, surrendering, volunteering, specific dogs.
+For more information having anything to do with Yankee Chihuahua Rescue and Adoption, Inc., please find your state below, click on it, and send a letter to your state coordinator. You may write concerning adoptions, surrendering, volunteering, or specific dogs.
 
 ...all the information you need is just a click away.
 