--- saved/bill-of-rights.txt
+++ live/bill-of-rights
@@ -4,7 +4,7 @@
 [IMAGE: im/logo/Yankee-Chi-Logo-transparent.png | alt="Yankee Chihuahua Rescue logo" | align= | width=211]
  | Serving the Six New England States
 
-[IMAGE: images/holidays/St%20Patrick/_happy%20st%20patricks%20day.png | alt="Happy St. Patrick's Day" | align= | width=298]
+[IMAGE: images/seasons/Spring/SpringSprung.jpg | alt="" | align= | width=381]
  | [LINK:i-give.html]iGive.com
 
 [LINK:happydogs/reconstruction.html]Happy Endings
@@ -47,7 +47,7 @@
 Ihave the right to be a full family member.
 [IMAGE: im/gr/r-chi5.png | alt="Chihuahua" | align= | width=90]
 
-Ineed to be in the house with you where I can learn how to behave. My sense of security and well being depends on being close to you.
+Ineed to be in the house with you, where I can learn how to behave. My sense of security and well-being depends on being close to you.
 Ihave the right to have an adult human to be my principal caretaker and teacher.
 Ihave the right to social stimulation.
 Ineed to go places of interest and learn new things and meet new people, so that I don't become fearful or aggressive - or depressed.
@@ -55,14 +55,14 @@
 
 [IMAGE: im/gr/r-chi1.png | alt="Chihuahua" | align= | width=86]
 Ihave the right to appropriate medical care.
-Ihave the right to regular exercise. I thrive on play and activity and it helps Me to be healthy and young.
-Ihave the right to be trained so that I do not become the prisoner of my misbehavior. Without training I can become confused, fearful and aggressive and become a danger to ourselves and to you! I am not happy unless I know what the rules are.
-Ihave the right to make mistakes. No matter how well we are trained or how old I get, sometimes I just do things wrong...it is nature. It won't happen often if I am properly trained.
-Ihave the right to be myself. My heritage influences who I am-my training helps shape us but some characteristics will remain. Please understand this when choosing me.
+Ihave the right to regular exercise. I thrive on play and activity, and it helps me to be healthy and young.
+Ihave the right to be trained so that I do not become a prisoner of my own misbehavior. Without training, I can become confused, fearful and aggressive and become a danger to myself and to you! I am not happy unless I know what the rules are.
+Ihave the right to make mistakes. No matter how well we are trained or how old I get, sometimes I just do things wrong... it is natural. It won't happen often if I am properly trained.
+Ihave the right to be myself. My heritage influences who I am -- training helps shape me but some characteristics will remain. Please understand this when choosing me.
 [IMAGE: im/gr/r-chi2.png | alt="Chihuahua" | align= | width=90]
 
-Ihave the right to move with my family. I am the only family member that can be rejected by current laws. I need you to fight for me and to change those laws so that the family can stay together.
-Ihave the right to live with dignity and to die with dignity. Please do not keep me alive for selfish reasons. Go with me on difficult journeys. Everything is easier for me if you are there.
+Ihave the right to move with my family. I am the only family member that can be rejected by current laws. I need you to fight for me and to change those laws, so that the family can stay together.
+Ihave the right to live with dignity and to die with dignity. Please, do not keep me alive for selfish reasons. Go with me on difficult journeys. Everything is easier for me if you are there.
 
 Remember -Ilove you!
 