--- saved/famous.txt
+++ live/famous
@@ -27,22 +27,22 @@
  | * Coco, featured in the television series Dog Whisperer and owned by Cesar Millan.
 * Dinky, the first Taco Bell Chihuahua.
 * Gidget, the second Taco Bell Chihuahua and the actress who played Bruiser's mom in Legally Blonde 2.
-* Moonie the actor who played Bruiser, Elle Woods' companion in the Legally Blonde movies.
+* Moonie, the actor who played Bruiser, Elle Woods' companion in the Legally Blonde movies.
 * Pepito, Xavier Cugat's Chihuahua in the children's book Pepito the Little Dancing Dog: The Story of Xavier Cugat's Chihuahua.
 * Tinkerbell and Bambi, companions of Paris Hilton. Tinkerbell is the "author" of "Tinkerbell Hilton Diaries".
 * Wheely Willy, a paraplegic Chihuahua from Long Beach, California who has become a celebrity as the subject of two bestselling children's books.
-* Lou, a Chihuahua that appears on every episode of The Soup on the E! network Fictional.
-* Chloe played by Angel, co-star of the film Beverly Hills Chihuahua.
+* Lou, a Chihuahua that appears on every episode of The Soup on the E! network.
+* Chloe, played by Angel, co-star of the film Beverly Hills Chihuahua.
 * Enrique, Hector Con Carne's pet Chihuahua from Evil Con Carne.
 * Geraldo, Max Shreck's Chihuahua in Batman Returns.
-* Madame Shirley the Medium, an old fortune-teller Chihuahua featured in a TV Show Cartoon, Courage the Cowardly Dog.
+* Madame Shirley the Medium, an old fortune-teller Chihuahua featured in a TV show cartoon, Courage the Cowardly Dog.
 * Mammoth Mutt, from the cartoon series Krypto the Superdog.
 * Mojo, Samuel Witwicky's Chihuahua from Transformers.
-* Papi played by Rusco, the other co-star of Beverly Hills Chihuahua.
-* Chip, Sonic's aide in Sonic Unleashed
+* Papi, played by Rusco, the other co-star of Beverly Hills Chihuahua.
+* Chip, Sonic's aide in Sonic Unleashed.
 * Ren Höek, of Ren and Stimpy fame.
 * The Spooky Chihuahua from Invader Zim.
-* Lilly Megan Hauserman's dog from I Love Money, Rock of Love Charm School, and Beauty and the Geek
+* Lilly, Megan Hauserman's dog from I Love Money, Rock of Love Charm School, and Beauty and the Geek.
  |
 
  |