--- saved/happy-endings.txt
+++ live/happy-endings
@@ -1,13 +1,11 @@
 [TABLE]
 
 |  |
-[IMAGE: ../im/logo/Yankee-Chi-Logo-transparent.png | alt="Yankee Chihuahua Rescue logo" | align= | width=212]
+[IMAGE: ../im/logo/Yankee-Chi-Logo-transparent.png | alt="Yankee Chihuahua Rescue logo" | align= | width=211]
  |
-[IMAGE: ../images/seasons/Chi-Sunflower.png | alt="Sunflower Chi" | align= | width=97]
-
-[IMAGE: ../images/seasons/summertime-chi3.png | alt="Ready for Vacation" | align= | width=195]
-
-[IMAGE: ../images/seasons/Chi-Flower.png | alt="Flower Chi" | align= | width=66]
+Serving the Six New England States
+
+[IMAGE: ../images/holidays/St%20Patrick/_happy%20st%20patricks%20day.png | alt="" | align= | width=298]
  | [LINK:../i-give.htm]iGive.com
 
 [LINK:../happydogs/reconstruction.html]Happy Endings
@@ -15,11 +13,14 @@
 [LINK:../events.html]Current Events
 
 [LINK:../guest/index.php]Guest Book
+
+[LINK:https://gofund.me/a6cb95fec]GoFundMe
 |  |
-[IMAGE: https://www.paypalobjects.com/en_US/i/scr/pixel.gif | alt="" | align= | width=1]
- | Serving the Six New England States
-
-[LINK:../index.html]
+[/TABLE]
+
+[TABLE]
+
+|  |  | [LINK:../index.html]
 [IMAGE: ../images/button68.gif | alt="Home" | align= | width=100]
 [LINK:../m-about.html]
 [IMAGE: ../images/button6B.gif | alt="About YCRAA" | align= | width=100]
@@ -31,14 +32,16 @@
 [IMAGE: ../images/button7A.gif | alt="Adoption" | align= | width=100]
 [LINK:../m-surrender.html]
 [IMAGE: ../images/button80.gif | alt="Surrender" | align= | width=100]
-
- | [LINK:../i-give.htm]
-[IMAGE: ../images/Save-a-Dog.gif | alt="" | align= | width=120]
-
+ |
 |  |
+[TABLE]
+
+|  |
 
 [IMAGE: ../images/pageunderconstruction-leftmargin.jpg | alt="Page Under Construction" | align= | width=115]
 
+|  |
+[/TABLE]
  |
 ####
 
@@ -817,15 +820,21 @@
 [IMAGE: happypics/01crop.jpg | alt="" | align= | width=150]
 
  |
+[TABLE]
+
+|  |
 
 [IMAGE: ../images/pageunderconstruction-leftmargin.jpg | alt="" | align= | width=135]
 
+|  |
+[/TABLE]
+
 |  |  |  |
-|  |
+|  |  | [LINK:http://www.igive.com/welcome/warmwelcome.cfm?c=47590]
+[IMAGE: ../images/iSearchiGive.gif | alt="iGive.com" | align= | width=486]
+[LINK:http://www.igive.com/html/refer.cfm?causeid=47590]
+[IMAGE: ../images/save-a-dog-logo-animated.gif | alt="iGive.com" | align= | width=120]
+ |
 [IMAGE: https://www.paypalobjects.com/en_US/i/scr/pixel.gif | alt="" | align= | width=1]
- |  | [LINK:http://www.igive.com/welcome/warmwelcome.cfm?c=47590]
-[IMAGE: ../images/iSearchiGive.gif | alt="iGive.com" | align= | width=486]
- |  | [LINK:http://www.igive.com/html/refer.cfm?causeid=47590]
-[IMAGE: ../images/save-a-dog-logo-animated.gif | alt="iGive.com" | align= | width=120]
 
 [/TABLE]