--- saved/info.txt
+++ live/info
@@ -28,24 +28,23 @@
 > AKC - Chihuahua Breed Standard
 > The Versatile Chihuahua
 > Chihuahua Bill of Rights
-> TeaCup Chihuahuas- Fact or Fiction?
-> For Fun,Famous Chihuahuas
+> Teacup Chihuahuas- Fact or Fiction?
+> For Fun:Famous Chihuahuas
 ### Articles by YCRAA Volunteers
 
 > YCRAA - The Beginningby Corinne Joly
 > Volunteeringby Cheryl
 ### Articles Related to Rescue and Adoption
 
-> Tips For New Adopters;by Anne Puchtinger, and Canadian Chihuahua Rescue & Transport
+> Tips For New Adopters: by Anne Puchtinger and Canadian Chihuahua Rescue & Transport
 > Are You a Responsible Pet Parent?
-> Benefits of Neutering- by Race Foster, DVM;Drs. Foster & Smith, Inc.
-> Why Rescues do not place dogs in homes with intact animals.
-
+> Benefits of Neutering- by Race Foster, DVM; Drs. Foster & Smith, Inc.
+> Why rescues do not place dogs in homes with intact animals
 ### Articles of Interest
 
 > Chihuahua Treats
 > Chocolate Poisoning
-> Crate training; step by step
+> Crate training: Step by Step
  |
 
 Our task must be to free ourselves... by widening our circle of compassion to embrace all living creatures and the whole of nature and its beauty.