--- saved/teacup.txt
+++ live/teacup
@@ -21,7 +21,7 @@
 |  |
 
  |
-# TeaCup Chihuahuas
+# Teacup Chihuahuas
 
 ## Fact or Myth?
 Sponsored by The Chihuahua Club of America
@@ -32,7 +32,7 @@
 
 As with all living things, there will be a size variance between individual dogs within this breed. Within the human family, brothers and sisters will differ in height and in weight, as well as other physical attributes. They are described as humans, male or female, and there is seldom if ever a need to break the description down further. The same holds true in regard to the Chihuahua; they are Chihuahuas-Long Coat/Smooth Coat, Male/Female.
 
-Unfortunately, the additional adjectives used to describe the size difference and physical appearances are many; and have been misused for so long they now seem legitimate. Tea-cup, Pocket Size, Tiny Toy, Miniature or Standard - are just a few of the many tags and labels that have been attached to this breed over the years. The Chihuahua Club of America is concerned that these terms may be used to entice prospective buyers into thinking that puppies described in this way are of greater monetary value. They are not; and the use of these terms is incorrect and misleading.
+Unfortunately, the additional adjectives used to describe the size difference and physical appearances are many; and have been misused for so long they now seem legitimate. Tea-cup, Pocket Size, Tiny Toy, Miniature, or Standard — are just a few of the many tags and labels that have been attached to this breed over the years. The Chihuahua Club of America is concerned that these terms may be used to entice prospective buyers into thinking that puppies described in this way are of greater monetary value. They are not; and the use of these terms is incorrect and misleading.
 
 Occasionally, within a litter, there may be a puppy that is unusually small. That puppy is a small Chihuahua and any other breakdown in description is not correct. To attach any of these additional labels to a particular pup is to misrepresent that animal as something that is rare or exceptional and causes a great deal of confusion among those fanciers who are looking for a Chihuahua.
 