--- saved/volunteer.txt
+++ live/volunteer
@@ -4,7 +4,7 @@
 [IMAGE: im/logo/Yankee-Chi-Logo-transparent.png | alt="Yankee Chihuahua Rescue logo" | align= | width=211]
  | Serving the Six New England States
 
-[IMAGE: images/holidays/St%20Patrick/_happy%20st%20patricks%20day.png | alt="Happy St. Patrick's Day" | align= | width=298]
+[IMAGE: images/seasons/Spring/SpringSprung.jpg | alt="" | align= | width=381]
  | [LINK:i-give.html]iGive.com
 
 [LINK:happydogs/reconstruction.html]Happy Endings
@@ -40,20 +40,18 @@
 
 |  |
 
-[IMAGE: images/holidays/St%20Patrick/Chi%20St%20Patrick%20pub2.png | alt="" | align= | width=207]
-
-[IMAGE: images/holidays/St%20Patrick/Chi%20St%20Patrick01.jpg | alt="" | align= | width=224]
+[IMAGE: images/seasons/Spring/Chi-DaisyPose.jpg | alt="" | align= | width=236]
 
  |
 ## Ever thought of volunteering for Chihuahua rescue?
 Volunteer opportunities are available in a variety of areas
 
 - monitoring shelters, notifying YCRAA about Chihuahuas that may be there
+[IMAGE: images/seasons/Spring/FlowerSmell.jpg | alt="" | align= | width=194]
+
 - providing permanent foster homes for special-needs Chihuahuas
-[IMAGE: images/holidays/St%20Patrick/ChiX2%20-%20happy%20st%20patrick%20day.jpg | alt="" | align= | width=226]
-
 - maintaining contact with shelters and other rescue groups
-- manning the YCRAA booth at fairs and shows
+- staffing the YCRAA booth at fairs and shows
 - screening applicants via telephoning and home visits
 - educating the public about the 'real' Chihuahua
 - creating and maintaining public relations and websites
@@ -64,18 +62,20 @@
 - maintaining records
 - responding to inquirers
 - fundraising
-- andmore!!
+- ... andmore!!
 #### Just a few hours can make a difference in the life of a homeless Chihuahua
 
 All these roles are critical to our success. Can you help?
 For more information email us:[LINK:mailto:volunteer@yankeechihuahuarescue.org?subject=I'd like more information about Volunteering]volunteer@yankeechihuahuarescue.org
 
+[IMAGE: images/Heart.png | alt="" | align= | width=200]
 [LINK:form/appi-volunteer-formspree.html]Volunteer Application[LINK:form/appi-foster-formspree.html]Foster Home Application
+[IMAGE: images/Heart.png | alt="" | align= | width=200]
 
-#### YCRAA objectives:
+#### YCRAA's objectives:
 
 - Rehabilitate and place homeless Chihuahuas and Chihuahua mixes in homes that meet the needs of each dog.
-- Promote responsible pet guardianship through education, resulting in a rewarding experience and relationship
+- Promote responsible pet guardianship through education, resulting in a rewarding experience and relationship.
 
 - How to choose the right dog.
 - Care for it.
@@ -85,7 +85,7 @@
 
 Each home is carefully screened before placement.
 
-Each dog is evaluated for temperament, has a vet exam, is up to date on all shots, and receives whatever other vet care is needed. Adult dogs are also spayed or neutered before adoption.
+Each dog is evaluated for temperament, has a vet exam, is up-to-date on all shots, and receives whatever other vet care is needed. Adult dogs are also spayed or neutered before adoption.
  |
 
 The basis of all animal rights should be the Golden Rule:
@@ -93,8 +93,6 @@
 We should treat them as we would wish them to treat us, were any other species in our dominant position.
 
 ~Christine Stevens
-
-[IMAGE: images/holidays/St%20Patrick/clover%20border%20right.png | alt="" | align= | width=115]
 
 |  |  | [LINK:http://www.igive.com/welcome/warmwelcome.cfm?c=47590]
 [IMAGE: images/iSearchiGive.gif | alt="iGive.com" | align= | width=460]