--- saved/volunteer.txt
+++ live/volunteer
@@ -25,10 +25,10 @@
 Volunteer opportunities are available in a variety of areas
 
 - monitoring shelters, notifying YCRAA about Chihuahuas that may be there
+
 - providing permanent foster homes for special-needs Chihuahuas
-
 - maintaining contact with shelters and other rescue groups
-- manning the YCRAA booth at fairs and shows
+- staffing the YCRAA booth at fairs and shows
 - screening applicants via telephoning and home visits
 - educating the public about the 'real' Chihuahua
 - creating and maintaining public relations and websites
@@ -39,7 +39,7 @@
 - maintaining records
 - responding to inquirers
 - fundraising
-- andmore!!
+- ... andmore!!
 #### Just a few hours can make a difference in the life of a homeless Chihuahua
 
 All these roles are critical to our success. Can you help?
@@ -47,10 +47,10 @@
 
 Volunteer ApplicationFoster Home Application
 
-#### YCRAA objectives:
+#### YCRAA's objectives:
 
 - Rehabilitate and place homeless Chihuahuas and Chihuahua mixes in homes that meet the needs of each dog.
-- Promote responsible pet guardianship through education, resulting in a rewarding experience and relationship
+- Promote responsible pet guardianship through education, resulting in a rewarding experience and relationship.
 
 - How to choose the right dog.
 - Care for it.
@@ -60,7 +60,7 @@
 
 Each home is carefully screened before placement.
 
-Each dog is evaluated for temperament, has a vet exam, is up to date on all shots, and receives whatever other vet care is needed. Adult dogs are also spayed or neutered before adoption.
+Each dog is evaluated for temperament, has a vet exam, is up-to-date on all shots, and receives whatever other vet care is needed. Adult dogs are also spayed or neutered before adoption.
  |
 
 The basis of all animal rights should be the Golden Rule: