--- saved/volunteering-cheryl.txt
+++ live/volunteering-cheryl
@@ -4,7 +4,7 @@
 [IMAGE: im/logo/Yankee-Chi-Logo-transparent.png | alt="Yankee Chihuahua Rescue logo" | align= | width=211]
  | Serving the Six New England States
 
-[IMAGE: images/holidays/St%20Patrick/_happy%20st%20patricks%20day.png | alt="Happy St. Patrick's Day" | align= | width=298]
+[IMAGE: images/seasons/Spring/SpringSprung.jpg | alt="" | align= | width=381]
  | [LINK:i-give.html]iGive.com
 
 [LINK:happydogs/reconstruction.html]Happy Endings
@@ -43,13 +43,13 @@
 
 by Cheryl “2009”
 
-In Nov of 2007, I had received an e-mail from the Chihuahua meet ups group of NH that caught my attention. It said something like “YCRAA is looking for a few good Volunteers!” The e-mail immediately grabbed my attention and I applied to volunteer. I had a home visit scheduled and was approved right away, I jumped in and got my first foster immediately. Then early 2008 it was uncertain if YCRAA would remain to rescue Chihuahua’s but in late spring 2008 we re-established and began rescuing again… Soon after the re start up, I was nominated as a Board of Director’s and gladly accepted the position..
+In Nov of 2007, I had received an e-mail from the Chihuahua meetups group of NH that caught my attention. It said something like “YCRAA is looking for a few good Volunteers!” The e-mail immediately grabbed my attention and I applied to volunteer. I had a home visit scheduled and was approved right away. I jumped in and got my first foster immediately. Then, in early 2008, it was uncertain if YCRAA would remain to rescue Chihuahuas, but in late spring 2008 we re-established and began rescuing again… Soon after the restart, I was nominated to the Board of Directors and gladly accepted the position.
 [IMAGE: im/gr/v-birtha.png | alt="Chihuahua" | align= | width=105]
 
-I am currently doing many home visits, both for volunteers and adoption applicants, meeting lots of wonderful people who enjoy the same interests, taking in dogs from shelters or owner surrenders, vet visits, fostering and placements.. Many people ask me "Isn't it hard to foster or to spend time with the dogs just to see them go?" and ironically, it's almost more rewarding. To get that first email from the new adopters knowing that the chi has made it through the night in their new homes and have their own "spot" is great, and many updates following how special the dog is to them, not to mention you get to "give another chi that second chance".
+I am currently doing many home visits, both for volunteers and adoption applicants, meeting lots of wonderful people who enjoy the same interests, taking in dogs from shelters or owner surrenders, vet visits, fostering and placements. Many people ask me "Isn't it hard to foster or to spend time with the dogs just to see them go?" and ironically, it's almost more rewarding. To get that first email from the new adopters knowing that each chi has made it through the night in its new home and has its own "spot" is great, and many updates following how special the dog is to them, not to mention you get to "give another chi that second chance".
 [IMAGE: im/gr/v-molly.png | alt="Chihuahua" | align= | width=107]
 
-Yankee Chihuahua Rescue is a wonderful and loving group of people who all have great compassion for Chihuahuas. There are so many Chi’s that would have been euthanized if it weren't for these volunteers who have provided their time, homes, blood sweat and tears to help these Chi-pups out. I have been lucky to meet some of the terrific dogs that would not be with us today if it were not for YCRAA. I am proud to be a member of this great organization and hope that we continue to attract more astonishing volunteers in the future. Volunteering for this rescue was the best decision I ever made.
+Yankee Chihuahua Rescue is a wonderful and loving group of people who all have great compassion for Chihuahuas. There are so many Chis that would have been euthanized if it weren't for these volunteers who have provided their time, homes, blood, sweat, and tears to help these Chi-pups out. I have been lucky to meet some of the terrific dogs that would not be with us today if it were not for YCRAA. I am proud to be a member of this great organization and hope that we continue to attract more astonishing volunteers in the future. Volunteering for this rescue was the best decision I ever made.
  |
 
 |  |  | [LINK:http://www.igive.com/welcome/warmwelcome.cfm?c=47590]