--- saved/why-no-intact.txt
+++ live/why-no-intact
@@ -21,11 +21,11 @@
 |  |  |
 ## Why we do not place dogs in homes with intact animals.
 
-The main reason why Yankee Chihuahua Rescue will not place a chihuahua in a home where the pets have not been neutered is because they deserve a home with responsible owners. Responsible owners take good care of their pets and do the best thing for them -- Neutering.
+The main reason why Yankee Chihuahua Rescue will not place a Chihuahua in a home where the pets have not been neutered is because they deserve a home with responsible owners. Responsible owners take good care of their pets and do the best thing for them — neutering.
 
 There are several health benefits to neutering:
 
-- One of the most important concerns the prostate gland, which under the influence of testosterone will gradually enlarge over the course of the dog's life. In age, it is likely to become uncomfortable, possibly being large enough to interfere with/defecation. The prostate under the influence of testosterone is also predisposed to infection which is almost impossible to clear up without neutering. Neutering causes the prostate to shrink into insignificance thus preventing both prostatitis as well as the uncomfortable benign hyperplasia (enlargement) that occurs with aging. It is often erroneously held that neutering prevents prostate cancer but this is not true.
+- One of the most important concerns the prostate gland, which under the influence of testosterone will gradually enlarge over the course of the dog's life. In age, it is likely to become uncomfortable, possibly being large enough to interfere with defecation. The prostate under the influence of testosterone is also predisposed to infection, which is almost impossible to clear up without neutering. Neutering causes the prostate to shrink into insignificance, thus preventing both prostatitis as well as the uncomfortable benign hyperplasia (enlargement) that occurs with aging. It is often erroneously held that neutering prevents prostate cancer, but this is not true.
 
 - Other health benefits of neutering include the prevention of certain types of hernias and tumors of the testicles and anus.
 
@@ -50,15 +50,15 @@
 
 |  |
 
-The only behavior changes after neutering relate to behaviors influenced by male hormones. Playfulness, friendliness, and socialization with humans are not changed. Activity level and appetite do not change with neutering. A male dog should not gain weight or become less interested in activity post neuter.
+The only behavior changes after neutering relate to behaviors influenced by male hormones. Playfulness, friendliness, and socialization with humans are not changed. Activity level and appetite do not change with neutering. A male dog should not gain weight or become less interested in activity post-neuter.
 
-- With Neutering, the interest in roaming is eliminated in 90 percent of neutered dogs
+- With neutering, the interest in roaming is eliminated in 90 percent of neutered dogs
 - Aggressive behavior against other dogs is eliminated in 60 percent of neutered dogs
 - Urine marking is eliminated in 50 percent of neutered male dogs
 - Inappropriate mounting is eliminated in 70 percent of neutered dogs
 - In other words, they are much better behaved.
 
-Well behaved companions are what a Yankee Chihuahua deserves.
+Well-behaved companions are what a Yankee Chihuahua deserves.
 
  |
 